### PR TITLE
Consent Screen

### PR DESCRIPTION
**Pull request by vanshg**
Story: https://app.shortcut.com/smileid/story/9097

## Summary

Adds the consent screen

### CODE REVIEW

File: Example/SmileID/Images.xcassets/SmileLogo.imageset/SmileLogo.pdf
Comment (jumaallan) [inline Example/SmileID/Images.xcassets/SmileLogo.imageset/SmileLogo.pdf:1]: Interesting that we can load pdf as files on iOS

Comment (vanshg) [reply Example/SmileID/Images.xcassets/SmileLogo.imageset/SmileLogo.pdf:1]: This is the preferred vector asset format for iOS (think XML vector asset on Android). SVG is supported as of Xcode 12, though. In both cases, Xcode generates PNGs for the respective graphics densities automatically.

**Review by jumaallan — COMMENTED** [review]
Can you see why the build fails

File: Example/SmileID/Images.xcassets/SmileLogo.imageset/SmileLogo.pdf
Comment (jumaallan) [reply Example/SmileID/Images.xcassets/SmileLogo.imageset/SmileLogo.pdf:1]: aah I see

**Review by jumaallan — APPROVED** [review]

**Pull request merged**